src/main/scala/cromwell/engine/backend/Backend.scala

```diff
@@ -85,6 +85,9 @@ trait Backend {
   @throws[IllegalArgumentException]("if a value is missing / incorrect")
   def assertWorkflowOptions(options: WorkflowOptions): Unit = {}
 
-  def makeTag(backendCall: BackendCall): String =
-    s"${this.getClass.getSimpleName} [UUID(${backendCall.workflowDescriptor.shortId}):${backendCall.call.name}]"
+  def makeTag(backendCall: BackendCall): String = {
+    val cls = this.getClass.getSimpleName
+    val clsString = if (cls.startsWith("anon")) "" else s"$cls "
+    s"$clsString[UUID(${backendCall.workflowDescriptor.shortId}):${backendCall.call.name}]"
```

Comment:
Just a minor annoyance where sometimes `this.getClass.getSimpleName` is sometimes (but rarely) something like `anon$1`.